osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/dlmsobjectconfig/DlmsObjectConfigServiceTest.java

```diff
@@ -79,28 +79,34 @@
     @Before
     public void setUp() {
         when(this.config422.contains(Protocol.DSMR_4_2_2)).thenReturn(true);
-        when(this.config422.findObject(DlmsObjectType.CLOCK, null)).thenReturn(Optional.of(clock1));
-        when(this.config422.findObject(DlmsObjectType.ACTIVE_ENERGY_IMPORT, null)).thenReturn(Optional.of(register));
-        when(this.config422.findObject(DlmsObjectType.MBUS_MASTER_VALUE, null)).thenReturn(Optional.of(registerWithChannel));
-        when(this.config422.findObject(DlmsObjectType.INTERVAL_VALUES, Medium.ELECTRICITY)).thenReturn(Optional.of(profileE));
-        when(this.config422.findObject(DlmsObjectType.DAILY_LOAD_PROFILE, null)).thenReturn(Optional.of(profileCombined));
-        when(this.config422.findObject(DlmsObjectType.DAILY_LOAD_PROFILE, Medium.GAS)).thenReturn(Optional.of(profileCombined));
-        when(this.config422.findObject(DlmsObjectType.DAILY_LOAD_PROFILE, Medium.ELECTRICITY)).thenReturn(Optional.of(profileCombined));
+        when(this.config422.findObject(DlmsObjectType.CLOCK, null)).thenReturn(Optional.of(this.clock1));
+        when(this.config422.findObject(DlmsObjectType.ACTIVE_ENERGY_IMPORT, null)).thenReturn(Optional.of(this.register));
+        when(this.config422.findObject(DlmsObjectType.MBUS_MASTER_VALUE, null)).thenReturn(Optional.of(
+                this.registerWithChannel));
+        when(this.config422.findObject(DlmsObjectType.INTERVAL_VALUES, Medium.ELECTRICITY)).thenReturn(Optional.of(
+                this.profileE));
+        when(this.config422.findObject(DlmsObjectType.DAILY_LOAD_PROFILE, null)).thenReturn(Optional.of(
+                this.profileCombined));
+        when(this.config422.findObject(DlmsObjectType.DAILY_LOAD_PROFILE, Medium.GAS)).thenReturn(Optional.of(
+                this.profileCombined));
+        when(this.config422.findObject(DlmsObjectType.DAILY_LOAD_PROFILE, Medium.ELECTRICITY)).thenReturn(Optional.of(
+                this.profileCombined));
```

Comment:
Above change looks serious but is only reformatting (introduction of `.this` by the onSave settings)